_docs/yasnippets/README.md

```diff
@@ -0,0 +1,21 @@
+# Yasnippets Collection
```

Comment:
MD022/blanks-around-headings/blanks-around-headers: Headings should be surrounded by blank

Reply:
Solved with: 98676aecedb3d6a9820c444b650f7910a8ff9dea.